specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/stable/2022-05-13/confidentialledger.json

```diff
@@ -77,9 +81,9 @@
         ],
         "responses": {
           "200": {
-            "description": "List of consortium members.",
+            "description": "Paged list of consortium members.",
             "schema": {
-              "$ref": "#/definitions/Consortium"
+              "$ref": "#/definitions/PagedConsortium"
```

Comment:
This doesn't look right.  This operation should be returning a (pageable) list of "members" -- not a pageable list of lists of members.

Rather than adding a new "PagedConsortium" schema, I think you should just add a `nextLink` to the `Consortium` schema.